lib/test.py

```diff
@@ -250,9 +251,12 @@ def run(self, server):
             color_stdout("[ updated ]\n", schema='test_new')
         else:
             has_result = os.path.exists(self.tmp_result)
+            result_checksum = ''
             if has_result:
                 shutil.copy(self.tmp_result, self.reject)
-            short_status = 'fail'
+                with open(self.tmp_result, mode='rb') as result_file:
+                    result_checksum = md5(result_file.read()).hexdigest()
+            short_status = 'fail:' + result_checksum
```

Comment:
I would pass it in some other way. Say, as a field in `WorkerTaskResult`. Because `short_status` now becomes a structure with a short status and a checksum, but it is not a structure, but a string… it is strange, at least. Not something that a reader of the code would expect.

Reply:
Corrected as you suggested.